### PR TITLE
Feature/pdalcol/misc updates

### PR DESCRIPTION
**Pull request by pedrodalcol**
# Critical Changes

# Changes
- Clarified help text for Hierarchy_Settings__c.Account_Processor__c.
- Corrected comment in ACCT_IndividualAccounts_TDTM.
- Updated @parameter annotations to @param.

# Issues Closed
- Fixes https://github.com/SalesforceFoundation/HEDAP/issues/512, https://github.com/SalesforceFoundation/HEDAP/issues/515, https://github.com/SalesforceFoundation/HEDAP/issues/516.
# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea
```
Comment (xinnanlisf) [inline .gitignore]: Question, does this have to be in the package?

Comment (pedrodalcol) [reply .gitignore]: It doesn't have to be in this pull request. @ceiroa suggested I commit it with my changes so we don't have to manually remove the .idea folder every time.

**Comment by xinnanlisf** [comment]
Looks good! Great job Pedro!

**Comment by pedrodalcol** [comment]
@lianeyee, can you review the release notes?

**Comment by annejackson** [comment]
Tested and verified the change. No issues found, hence merging the Pr.

**Pull request merged**